syft/execution/computation.py

```diff
@@ -201,3 +202,32 @@ def unbufferize(
             action = ComputationAction(command, target, tuple(args_), kwargs_, tuple(return_ids))
 
         return action
+
```

Comment:
👍 

This seems like the kind of thing that could live in `Action`, so that it can be shared across `Action` types.

Reply:
I checked CommunicationAction, it seems to have different structure.
If we'll have such method in Action, it will need to distinguish between communication/computation.
It's probably better for communication to have own method?

Reply:
The structure is different, but they should at least partially converge. This method doesn't seem to be very structure dependent though, other than the list of attributes to `stringify`. Seems like there could be a common, general method backing them both, maybe with args to specify which fields to `stringify` or something.

Could happen here or be deferred for a later PR. 🤷